### PR TITLE
fix(AI): A data dictionary entry was in the wrong folder

### PR DESCRIPTION
**Pull request by paperclypse**
I needed a draft PR so I can see if this change fixes this.

### CODE REVIEW

**Comment by paperclypse** [comment]
This data dictionary entry was broken, because it didn't have its own folder.

**Comment by urbiz-nr** [comment]
@paperclypse - I've slacked you my feedback.

File: src/data-dictionary/events/NrAiIncidentExternal/aggregationTag.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: aggregationTag.*
+type: attribute
+events:
+  - NrAiIncidentExternal
+---
+
+Any trigger events with the `aggregationTag` prefix and the same suffix attribute are grouped together in an incident. A resolve event resets this aggregation.
```
Comment (bradleycamacho) [inline src/data-dictionary/events/NrAiIncidentExternal/aggregationTag.md:8]: This seems a bit confusing to me, but I may just not have enough context. I'm confused by what "same suffix" means here. Would it be `aggregationTag.triggerHere.aggregationTag`?

File: src/data-dictionary/events/NrAiIncidentExternal/entity.guid.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: entity.guid
+type: attribute
+events:
+  - NrAiIncidentExternal
+---
+
+The identifier of the entity linked to the incident.
```
Comment (bradleycamacho) [inline src/data-dictionary/events/NrAiIncidentExternal/entity.guid.md:8]: Maybe define `GUID` here?

File: src/data-dictionary/events/NrAiIncidentExternal/state.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+name: state
+type: attribute
+events:
+  - NrAiIncidentExternal
+---
+
+Whether the event should trigger a new incident or resolve an existing one. Enum: `trigger`, `resolve`, and `acknowledge`. Required.
```
Comment (bradleycamacho) [inline src/data-dictionary/events/NrAiIncidentExternal/state.md:8]: I'm not sure what `enum` is so I'm going to flag it, but if it's something users would know ignore this.

File: src/data-dictionary/events/NrAiIncidentExternal/tag.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+name: tag.*
+type: attribute
+events:
+  - NrAiIncidentExternal
+---
+
+Arbitrary key-value metadata, or tags, associated with the violation. `tag.` is the prefix and `*` is the metadata/tag name.
```
Comment (bradleycamacho) [inline src/data-dictionary/events/NrAiIncidentExternal/tag.md:8]:
```suggestion
Arbitrary key-value metadata, or tags, associated with the violation. `tag` is the prefix and `*` is the metadata/tag name.
```

Comment (bradleycamacho) [reply src/data-dictionary/events/NrAiIncidentExternal/tag.md:8]: I believe this is correct, but I'm unsure haha

**Review by bradleycamacho — APPROVED** [review]
Approved pending a few small changes. Most are small so I give you free reign on them.

File: src/data-dictionary/events/NrAiIncidentExternal/aggregationTag.md (hunk 1 above)
Comment (paperclypse) [reply src/data-dictionary/events/NrAiIncidentExternal/aggregationTag.md:8]: I think I came up with something clearer.

**Pull request merged**